### ISSUE TITLE
Test set missing elements

### ISSUE THREAD

**Issue by eschkufz** [StanfordPL/stoke#16]
Stoke search won't use testcases that have been used for training for regression testing. So if test set == training set, test set will be empty.
